slopmop/utils/environment.py

```
@@ -0,0 +1,40 @@
+    if os.environ.get("GEMINI_CLI"):
+        return True
+    if os.environ.get("CLAUDE_CODE"):
+        return True
+    if os.environ.get("AGENT_MODE"):
+        return True
+    # TERM_PROGRAM is often set by specialized terminal emulators or agents
+    term_program = os.environ.get("TERM_PROGRAM", "")
+    if term_program in ("Gemini", "ClaudeCode"):
+        return True
+
+    return False
```

Ruff (0.15.15)

[warning] Return the condition `term_program in ("Gemini", "ClaudeCode")` directly

Replace with `return term_program in ("Gemini", "ClaudeCode")`

(SIM103)